framework/TApplicationConfiguration.php

```diff
@@ -234,6 +250,60 @@ protected function loadPathsXml($pathsNode, $configPath)
 			}
 		}
 	}
+	
+	/**
+	 * Reads the Composer static RegisteredLoaders for their Vendor Directory. Reads the Vendor
+	 * Directory composer file 'installed.json' (accumulated extensions composer.json) for the project.
+	 * The ['extra']['bootstrap'] field is read for each extension, if it's there.
+	 * @return array<string, string> the extension name and bootstrap class.
+	 * @since 4.2.0
+	 */
+	protected function getComposerExtensionBootStraps()
+	{
+		if ($cache = Prado::getApplication()->getCache()) {
+			$plugins = $cache->get(self::COMPOSER_INSTALLED_CACHE);
+			if ($plugins !== null) {
+				return $plugins;
+			}
+		}
+		$dependencies = new TChainedCacheDependency();
+		$listDeps = $dependencies->getDependencies();
+		$plugins = [];
+		foreach (ClassLoader::getRegisteredLoaders() as $vendorDir => $loader) {
+			echo($vendorDir . "\n");
```

Comment:
Remove debug, or change it to a Prado::trace() if useful

Reply:
oops.  my bad.  I'll get on this.

Reply:
I'm so glad you like this composer integration PR.  It's sorely needed for the community.  This is modeled on Yii.